metadata.py

```diff
@@ -16,15 +16,17 @@
 import sys
 import os
 from astropy.io import fits
-from common import make_dir_md5_table
 import datetime
 import re
 import pandas as pd
 import html
+import pdb
+import glob
+import gzip
+import hashlib
+import logging
```

Comment:
From my test, the logging in this file does not get added to the main output log file that is defined in creae_log.py.  I did a quick test, and I think we can solve this by getting the logging object by the same name as it was generated in create_log.py.  So I put this at the top of metadata.py in the module global scope and changed the "logging." calls to "log.":

`
log = logging.getLogger(f"dep <{os.getlogin()}>")
`

Reply:
This will be addressed and tested today. Thanks for spotting it.